tests/integration/test_reduction_complex.py

```diff
@@ -13,25 +13,25 @@
 # limitations under the License.
 #
 
-import random
-
 import numpy as np
+import pytest
 
 import cunumeric as num
 
+x = np.array([1 + 4j, 2 + 5j, 3 + 6j], np.complex64)
 
-def test():
-    anp = np.random.randn(4, 5)
-    b = random.randint(1, 13)
-    a = num.array(anp)
 
-    np.equal(anp, b, out=anp)
-    num.equal(a, b, out=a)
+def test_sum():
```

Comment:
Ok. this looks like another incorrect matching of files.

Reply:
Same here, I think there was some mix up in one of the intermediate commits, but the final result matches the current test:

https://github.com/nv-legate/cunumeric/blob/branch-22.05/tests/reduction_complex.py

Reply:
looks good, though I don't know why this test wasn't calling `np.allclose`.